pkg/util/crd_defaults.go

```diff
@@ -652,7 +652,7 @@ func GetAlertDefault() *alertv1.AlertSpec {
 		Port:                 &port,
 		EncryptionPassword:   "",
 		EncryptionGlobalSalt: "",
-		PersistentStorage:    true,
+		PersistentStorage:    false,
```

Comment:
Synopsysctl uses this template when generating Alert custom resources. The persistent storage for that one should remain true